content/en/docs/tasks/configure-pod-container/static-pod.md

```diff
@@ -132,18 +132,18 @@ and store it on a web server so that you can pass the URL of that file to the ku
 
 Configure the kubelet on your selected node to use this web manifest by running it with `--manifest-url=<manifest-url>`
 
-    On Fedora, edit `/etc/kubernetes/kubelet` to include this line:
-
-    ```
-    KUBELET_ARGS="--cluster-dns=10.254.0.10 --cluster-domain=kube.local --manifest-url=<manifest-url>`
-    ```
+> On Fedora, edit `/etc/kubernetes/kubelet` to include this line:
+>
+> ```
+> KUBELET_ARGS="--cluster-dns=10.254.0.10 --cluster-domain=kube.local --manifest-url=<manifest-url>`
+> ```
```

Comment:
What are the above ">" characters?

Reply:
They are a blockquote tag of Markdown.

There are two guides for Markdown which explains the syntax.

* [Basic Syntax | Markdown Guide](https://www.markdownguide.org/basic-syntax/#blockquotes-1)
* [Mastering Markdown · GitHub Guides](https://guides.github.com/features/mastering-markdown/)

I review the render result, and I found the tool of Markdown maybe have a bug.

## Before I fix the error:

<https://kubernetes.io/docs/tasks/configure-pod-container/static-pod/>

![image](https://user-images.githubusercontent.com/2121467/65403972-1fbc0b00-de09-11e9-9889-b2b40f8706a0.png)


### After:

<https://deploy-preview-16452--kubernetes-io-master-staging.netlify.com/docs/tasks/configure-pod-container/static-pod/>

![image](https://user-images.githubusercontent.com/2121467/65403946-f307f380-de08-11e9-9e39-fa6837c43085.png)

### Right render

But it should render like as following (Preview on VS Code):

![image](https://user-images.githubusercontent.com/2121467/65403938-dff52380-de08-11e9-85b1-0f2ee28e2d01.png)